packages/react/src/utils/styles.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export const STYLE_DISABLE_SCROLLBAR = {
```

Comment:
Let's not overuse all caps. It doesn't matter much for consumers that it's a constant.

```suggestion
export const disableScrollbarStyle = {
```